web/src/invite.ts

```diff
@@ -451,10 +411,65 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
         };
 
         $("#invite-user-form .setup-tips-container").html(render_invite_tips_banner(context));
+
+        const toggler = components.toggle({
+            html_class: "invite_users_option_tabs",
+            selected: 0,
```

Comment:
Does this work correctly when only the second tab is non-disabled?

Reply:
It does, because when email (first tab) is disallowed, it toggles to the second tab.

Reply:
Passing this is actually not needed if you are handling toggling to the second tab if first is disabled below by calling `toggler.goto` as the first tab will be opened by default anyways.